test/integration/users.test.js

```diff
@@ -350,6 +350,134 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users?search", function () {
+    const userSearchParamsData = [
+      {
+        username: "ankur",
+        first_name: "Ankur",
+        last_name: "Narkhede",
+        yoe: 0,
+        img: "./img.png",
+        linkedin_id: "ankurnarkhede",
+        github_id: "ankur1234",
+        github_display_name: "ankur-xyz",
+        phone: "1234567890",
+        email: "abc@gmail.com",
+      },
+      {
+        username: "23ankur",
+        first_name: "Ankur",
+        last_name: "Narkhede",
+        yoe: 0,
+        img: "./img.png",
+        linkedin_id: "ankurnarkhede",
+        github_id: "ankur1234",
+        github_display_name: "ankur-xyz",
+        phone: "1234567890",
+        email: "abc@gmail.com",
+      },
+    ];
+
+    beforeEach(async function () {
+      await addOrUpdate(userSearchParamsData[0]);
+      await addOrUpdate(userSearchParamsData[1]);
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+    });
+
+    it(`${searchParamValues[0].desc}`, function (done) {
```

Comment:
I don't know what the test does here, from its name.
We should write the test name, I feel it's just confusing to maintain a separate file for test names.

Reply:
done